tests/python/contrib/test_hexagon/metaschedule_e2e/test_resnet50_int8.py

```diff
@@ -372,6 +596,11 @@ def test_packed_8x8x32_resnet50(hexagon_launcher):
 
         np.testing.assert_allclose(ref_result, hexagon_output, atol=1e-4, rtol=1e-5)
 
+        debug_ex = session.get_graph_debug_executor(
+            hexagon_lowered.get_graph_json(), hexagon_lowered.lib
+        )
+        print(debug_ex.profile(input_name=inp.copy()))
```

Comment:
Do we want `benchmark` at L589 and profiling here, given that this test is not really tuning anything?

Reply:
I wanted to run these to see what the performance of the model is in summation after we use all of these schedules. Is there a better way to get that?